src/plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -94,6 +94,8 @@ export function getTimelionRequestHandler({
       });
 
     try {
+      const searchSessionOptions =
+        searchSessionId && dataSearch.session.getSearchOptions(searchSessionId);
```

Comment:
`getSearchOptions` can return `null` now. timelion route validation fails in case we send
`searchSession: null`, so this change just makes sure `searchSession` key is not send in case `getSearchOptions` returns null